specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2022-03-03/examples/galleryExamples/Gallery_AddToSharingProfile.json

```diff
@@ -46,7 +46,7 @@
     },
     "202": {
       "headers": {
-        "Location": "https://management.azure.com/subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.Compute/galleries/myGalleryName/share?api-version=2022-01-03"
+        "Location": "https://management.azure.com/subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.Compute/galleries/myGalleryName/share?api-version=2022-03-03"
```

Comment:
This file and the 2 others beneath it were not in the original PR, https://github.com/Azure/azure-rest-api-specs/pull/19644/files#diff-3a9a5c0c9286b79888f07bb40f5028aab4f7765107805633be820010fd1e34c6L2633 , but their inclusion makes sense. 
Is this difference expected?

Reply:
Tim said this is expected, and I would agree.